src/api/operator/accountOrderUtils.ts

```diff
@@ -1,7 +1,8 @@
-import { OrderMetaData, SupportedChainId } from '@cowprotocol/cow-sdk'
-import { COW_SDK } from 'const'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
 import { GetAccountOrdersParams, RawOrder } from './types'
+import { orderBookSDK } from 'cowSdk'
+import { EnrichedOrder } from '@cowprotocol/cow-sdk/order-book'
```

Comment:
these types are not exported fro the main entry point just to avoid collisions, right?

Reply:
Yes